import-service/src/functions/importFileParser/handler.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  S3Client,
+  GetObjectCommand,
+  CopyObjectCommand,
+  DeleteObjectCommand,
+} from '@aws-sdk/client-s3';
+import csv from 'csv-parser';
+
+import { formatJSONResponse } from '@libs/apiGateway';
+import { middyfy } from '@libs/lambda';
+import stream from 'stream';
```

Comment:
You can import promisified version of all stream from `stream/promises`. Described [here](https://nodejs.org/api/stream.html#streams-promises-api).

Reply:
Unfortunately, it works only from version 15.0.0 of NodeJs
https://nodejs.org/api/stream.html#streams-promises-api

Reply:
You are right; I've missed it. We are using v16 on the project 🙂 .